nntrainer/tensor/manager.cpp

```diff
@@ -918,6 +938,37 @@ void Manager::expand_lifespan(const std::string &name,
     enum_class_or<TensorLifespan>(tensor_lifespan_map[name], lifespan);
 }
 
+/**
+ * @brief     Create tensors with the given spec
+ */
+std::vector<Var_Grad *> Manager::requestAllocatedOutputsAsInputs(
```

Comment:
allocated sounds like memory is allocated. may be just `requestOutputsAsInputs`? or have different word than allocated?

Reply:
This is merged to `requestInputs()` after 2 more PRs, so the name can be ignored here.